src/index.js

```diff
@@ -64,7 +63,6 @@ function App() {
         <Route path="/menteeapplication" component={Mentee} />
         <Route path="/mentorapplication" component={Mentor} />
         <Route path="/login" component={LoginPage} />
-        <Route path="/pendingapproval" component={PendingApproval} />
         <Route path="/availability" component={Availability} />
```

Comment:
Would we be placing these routes, including this new pending approval in a Secure Route?